source/Playnite/Localization/LocSource.xaml

```diff
@@ -372,8 +372,8 @@ New database location will only be used after application is restarted.</sys:Str
     <sys:String x:Key="LOCSteamBackgroundSourceScreenshot">Store Screenshot</sys:String>
     <sys:String x:Key="LOCSteamBackgroundSourceStore">Store Background</sys:String>
     <sys:String x:Key="LOCBackgroundImageScreenOptionTooltip" xml:space="preserve">Doesn't apply retrospectively to existing games without re-downloading metadata.</sys:String>
-    <sys:String x:Key="LOCSettingsForceDownloadPlaynite">Force sync play time</sys:String>
-    <sys:String x:Key="LOCSettingsForceDownloadPlayniteTooltip">Always download recorded play time from remote library during library sync, ignoring time recorded by Playnite.</sys:String>
+    <sys:String x:Key="LOCSettingsForceDownloadPlaynite">Force sync Play Time for supported libraries</sys:String>

```

Comment:
I like the addition of "supported library". Could you also change it from "Force sync" to "Prioritize"? I think that would be a better description.

Reply:
So from `Force sync Play Time for supported libraries` to `Prioritize play time of supported libraries`? I'm not too sure myself if that string is self-explanatory enough to say what it does, in my opinion the word "sync" shouldn't be removed. I have a few suggestions, tell me what you think or if you still prefer your suggestion:

1. Synchronize play time of supported libraries
2. Synchronize play time of supported libraries on game library update
3. Synchronize play time of supported libraries when updating game libraries

I see there's a mention of "library during library sync" in the tooltip, but in my opinion that information goes better in the option name itself.

I'll be modifying the string to your suggestion for the moment but tell me if you like some of my suggestions.

Reply:
Got it, everything is modified exactly as you suggested.